bmicalc/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Jun 23 08:50:38 CEST 2017
+distributionBase=GRADLE_USER_HOME

```

Comment:
what the ^M here ?

Reply:
don't know why it is coming but it is not showing in my editor I've pushed it once again can you please check

Reply:
I guess it won't create any problem